### PR TITLE
Add integration tests for <amp-analytics type=googleanalytics>

### LINKED ISSUE #9526: Add integration test for CID logic that involves cookies
Which can prevent bug #9524 from happening

### LINKED ISSUE #19708: Add integration test for <amp-analytics type=googleanalytics>

### PR DESCRIPTION
**Pull request by lannka**
Closes #19708

### CODE REVIEW

File: test/integration/test-amp-analytics.js [hunk 1]
```diff
@@ -190,6 +191,155 @@ describe.configure().skipIfPropertiesObfuscated().run('amp' +
...
+        document.cookie='_ga=;expires=' + new Date(0).toUTCString();
+      </script>
+      <amp-analytics type="googleanalytics">
+        <script type="application/json">
+        {
+          "vars": {
+            "account": "UA-67833617-1"
+          },
+          "requests": {
+            "host": "${RequestBank.getUrl(1)}"
```
Comment (calebcordry) [inline test/integration/test-amp-analytics.js]: I thought we were going to name these?

File: test/integration/test-amp-analytics.js [hunk 2]
```diff
@@ -190,6 +191,155 @@ describe.configure().skipIfPropertiesObfuscated().run('amp' +
...
+      </amp-analytics>
+      `,
+    extensions: ['amp-analytics'],
+  }, () => {
+    afterEach(() => {
+      // clean up written _ga cookie
+      document.cookie = '_ga=;expires=' + new Date(0).toUTCString();
+    });
+
+    it('should assign new cid', () => {
```
Comment (zhouyx) [inline test/integration/test-amp-analytics.js]: Do we have test to get stored cid? Or is that not possible to do.

Comment (lannka) [reply test/integration/test-amp-analytics.js]: the test above is parsing CID from cookie

Comment (zhouyx) [reply test/integration/test-amp-analytics.js]: Missed that. Thanks.

**Comment by lannka** [comment]
Decided to move the CLIENT_ID related tests out of this googleanalytics specific tests #19709

**Pull request merged**